apps/www/components/Article/Page.js

```diff
@@ -69,20 +65,21 @@ import SectionNav from '../Sections/SinglePageNav'
 import SectionFeed from '../Sections/SinglePageFeed'
 import HrefLink from '../Link/Href'
 import { withMarkAsReadMutation } from '../Notifications/enhancers'
-import { cleanAsPath } from '../../lib/utils/link'
-import { getMetaData, runMetaFromQuery } from './metadata'
-import FlyerFooter, { FlyerMeta, FlyerNav } from './Flyer'
-import ShareImageFlyer from './ShareImageFlyer'
-import { getFlyerTileActionBar } from '../ActionBar/FlyerTileActionBar'
+import ShareImageFlyer from '../Flyer/ShareImage'
+import FlyerPage from '../Flyer'
```

Comment:
I find it confusing that the import form index is called FlyerPage, I'd expect it to be called Flyer

Reply:
done